scripts/versions.sh

```diff
@@ -6,4 +6,4 @@
 set -euo pipefail
 
 # Don't export them as they're used in the context of other calls
-AVALANCHE_VERSION=${AVALANCHE_VERSION:-'54d8b06b'}
+AVALANCHE_VERSION=${AVALANCHE_VERSION:-'55b6a3fa9'}
```

Comment:
This needs to be 8 characters, not 9 😉 

```suggestion
AVALANCHE_VERSION=${AVALANCHE_VERSION:-'55b6a3fa'}
```